core/src/com/unciv/logic/civilization/ReligionManager.kt

```diff
@@ -9,13 +11,13 @@ class ReligionManager {
 
     var storedFaith = 0
 
-    var pantheonBelief: String? = null
+    var religion: Religion? = null
```

Comment:
Religions shouldn't be stored both in civs and in game info.
There should be only one source of truth for each religion